juniper_official/Linecard/cm-events.rule

```diff
@@ -159,6 +159,26 @@ iceberg {
                 description "CM error threshold: Number of errors increase between metrics, before we report anomaly";
                 type int;
             }
+            rule-properties {
+                version 2;
+                contributor juniper;
+                supported-healthbot-version 1.0.1;
+                catalogue {
+                    tier 1;
+                }
+                supported-devices {
+                    juniper {
+                        operating-system junos {
+                            products MX {
+                                releases 17.2 {
+                                    release-support min-supported-release;
+                                    platform All-i386;
```

Comment:
"All-i386" is not a platform entry..  Please have a valid list of platform name. Example MX120, MX2020. This will be used in the future releases and matched against the device-facts